app/src/components/v-form/v-form.vue

```diff
@@ -161,8 +162,21 @@ function useForm() {
 		return fields;
 	});
 
+	const fieldDetailStore = useFieldDetailStore();
+
 	const fieldsMap = computed<Record<string, TFormField | undefined>>(() => {
-		return Object.fromEntries(fieldsWithConditions.value.map((field) => [field.field, field]));
+		const fieldsObject = Object.fromEntries(fieldsWithConditions.value.map((field) => [field.field, field]));
+
+		// Set the fields of Maximum and Minimum values on the interface of a table field
+		// to accept bigInteger values if the field itself is a bigInteger
+		if (fieldsObject['min'] && fieldsObject['max']) {
+			if (fieldDetailStore.field.type && fieldDetailStore.field.type == 'bigInteger') {
+				fieldsObject['min'].type = 'bigInteger';
+				fieldsObject['max'].type = 'bigInteger';
+			}
+		}
```

Comment:
This should be handled in the interface responsible for handling min/max on big integers on in the globally used v-form container. 
```suggestion
```